podvm-builder/podvm-builder.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+
+# function to create podvm image
+
+function create_podvm_image() {
+  case "${CLOUD_PROVIDER}" in
+  azure)
+    echo "Creating Azure image"
+    /scripts/azure-podvm-image-handler.sh -c
+    if [ "${UPDATE_PEERPODS_CM}" == "yes" ]; then
+      # Get the IMAGE_ID value from the podvm-images configmap
+      # key: azure and the first value in the list
+      IMAGE_ID=$(kubectl get configmap podvm-images -n openshift-sandboxed-containers-operator -o jsonpath='{.data.azure}' | awk '{print $1}')
+      # Update peer-pods-cm configmap with the IMAGE_ID value
+      echo "Updating peer-pods-cm configmap with IMAGE_ID=${IMAGE_ID}"
+      kubectl patch configmap peer-pods-cm -n openshift-sandboxed-containers-operator --type merge -p "{\"data\":{\"AZURE_IMAGE_ID\":\"${IMAGE_ID}\"}}"
+    fi
+    ;;
+  aws)
+    echo "Creating AWS AMI"
+    /scripts/aws-podvm-image-handler.sh -c
+    if [ "${UPDATE_PEERPODS_CM}" == "yes" ]; then
+      # Get the AMI_ID value from the podvm-images configmap
+      # key: aws and the first value in the list
+      AMI_ID=$(kubectl get configmap podvm-images -n openshift-sandboxed-containers-operator -o jsonpath='{.data.aws}' | awk '{print $1}')
+      # Update peer-pods-cm configmap with the AMI_ID value
+      echo "Updating peer-pods-cm configmap with AMI_ID=${AMI_ID}"
+      kubectl patch configmap peer-pods-cm -n openshift-sandboxed-containers-operator --type merge -p "{\"data\":{\"PODVM_AMI_ID\":\"${AMI_ID}\"}}"
+    fi
+    ;;
+  *)
+    echo "CLOUD_PROVIDER is not set to azure or aws"
+    exit 1
+    ;;
+  esac
+}
+
+# Function to delete podvm image
+function delete_podvm_image() {
+  case "${CLOUD_PROVIDER}" in
+  azure)
+    echo "Deleting Azure image"
+    /scripts/azure-podvm-image-handler.sh -C
+    # Update the podvm-images configmap to remove the azure image id from the list
+    # Get the IMAGE_ID_LIST from the podvm-images configmap
+    IMAGE_ID_LIST=$(kubectl get configmap podvm-images -n openshift-sandboxed-containers-operator -o jsonpath='{.data.azure}')
+
+    # Remove the IMAGE_ID from the list
+    IMAGE_ID_LIST="${IMAGE_ID_LIST//${IMAGE_ID}/}"
+
+    # Update the podvm-images configmap with the new list
+    kubectl patch configmap podvm-images -n openshift-sandboxed-containers-operator --type merge -p "{\"data\":{\"azure\":\"${IMAGE_ID_LIST}\"}}"
```

Comment:
maybe support UPDATE_PEERPODS_CM also here?

Reply:
Hmm yeah, in case if the IMAGE is the same in the peer-pods-cm. Will also need to create a warning.
Instead I think for delete image, I should give a warning if the image to be deleted is configured in peer-pods-cm and provide a force option.
Wdyt ?

Reply:
For deletion, admin must provide the the IMAGE_ID or AMI_ID, otherwise it'll error out. This is a safety mechanism to avoid assuming whether to delete the latest or the oldest.

Reply:
In the readme I have added instructions on how to get the IMAGE_ID or AMI_ID for deletion

Reply:
maybe worth add **IMAGE_ID: ""** as template in the CM

Reply:
Sorry.. I didn't understand this. Can you please check the README and suggest if the instruction will suffice? 
We'll need to provide detailed docs for creation and deletion and that should cover different scenarios.